swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        if item in self.inventory:
```

Comment:
If we got past the guard clause, we know that item _must_ be in the inventory. Do we need this additional check?

And is there a way we could call `remove` directly with no check at all? How can we tell whether `remove` fails?